roteiro-de-estudos/README.md

```diff
@@ -0,0 +1,37 @@
+# Roteiro de Estudos
+
+Na trajetória de um maratonista em programação competitiva, muitos tópicos de computação serão necessários para chegar cada vez mais longe, e embora tenha tanta coisa que seja praticamente impossível dominar tudo, uma base sólida deve ser construída a fim de desenvolver ainda mais o potencial destes maratonistas.
+
+Um dos objetivos do GEMP é de **curadoria** de conteúdo, ou seja, queremos que as principais referências de estudo sejam as melhores. Valorizamos fontes didáticas, com boa qualidade na explicação, de forma que menos tempo seja gasto procurando conteúdo minimamente aproveitável e mais tempo seja investido estudando, praticando ou discutindo com a comunidade.
+
+A seguir estão os principais tópicos de estudo aos quais estaremos incluindo referências e comentários.
+
+## Ciclo Básico
+
+0. Programação Básica
+1. Análise de Algoritmos
+2. Estruturas de Dados
+3. Paradigmas de Resolução de Problemas
+4. Teoria dos Grafos
+5. Matemática Computacional
+6. Algoritmos de Processamento de Texto
+
+## Visão Geral
+
+Tudo começa com o aprendizado de **programação básica**, onde nos familiarizamos com linguagens de programação, sua documentação e seus principais recursos para construir soluções, além de trabalhar aspectos iniciais de *problem-solving*, como testar, debugar e discutir nossas soluções, como também de internalizar que problemas podem ser resolvidos de várias formas diferentes, algumas melhores que outras.
```

Comment:
Como o nosso texto é dedicado para leitoras(es) leigas, talvez seja interessante termos um README dedicado a explicar o que é um problema de programação competitiva.

Reply:
Eu não diria que nosso público alvo é necessariamente leigo, pelo menos não nesse momento que ainda tem muita coisa faltando. Digo, por enquanto não temos um planejamento de fazer tudo desde o começo, pedacinho por pedacinho, e sim ir preenchendo a medida que formos tocando nos tópicos, sempre que for necessário. 

Mas concordo que ter uma explicação sobre que tipos de problema e o escopo bem delimitado em que estudamos/competimos seria uma boa adição. Não acha que nesse ponto seria melhor criar uma issue relacionada a criação desse documento mais explanatório? De fato, ter alguns arquivos pequenos como *quick notes* é interessante.

Reply:
Uma Issue foi criada pra tratar disso #15